crates/turborepo-auth/src/auth/sso.rs

```diff
@@ -45,7 +51,30 @@ pub async fn sso_login<'a>(
             {
                 println!("{}", ui.apply(BOLD.apply_to("Existing token found!")));
                 ui::print_cli_authorized(&response_user.user.email, ui);
-                return Ok(token.into());
+                return Ok(Token {
+                    token: token.into(),
+                    exists: true,
+                });
+            }
+        }
+    }
+
+    // No existing token found. If the user is logging into Vercel, check for an
+    // existing `vc` token with correct scope.
+    if login_url_configuration.contains("vercel.com") {
```

Comment:
This is the same in `login`, but I plan on refactoring this in another PR since I already did a good amount of refactoring in this one.